validator/client/validator_aggregate_test.go

```diff
@@ -78,3 +79,26 @@ func TestWaitForSlotTwoThird_WaitCorrectly(t *testing.T) {
 		t.Errorf("Wanted %d time for slot two third but got %d", twoThirdTime, currentTime)
 	}
 }
+
+func TestAggregateAndProofSignature_CanSignValidSignature(t *testing.T) {
+	validator, m, finish := setup(t)
+	defer finish()
+
+	m.validatorClient.EXPECT().DomainData(
+		gomock.Any(), // ctx
+		gomock.Any(), // epoch
+	).Return(&ethpb.DomainResponse{}, nil /*err*/)
+
+	agg := &ethpb.AggregateAttestationAndProof{
+		AggregatorIndex: 0,
+		Aggregate:       &ethpb.Attestation{Data: &ethpb.AttestationData{}},
+		SelectionProof:  nil,
+	}
+	sig, err := validator.aggregateAndProofSig(context.Background(), validatorPubKey, agg)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if _, err := bls.SignatureFromBytes(sig); err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Can you add a sig verify or something for this?

Reply:
This is sufficient. We just want to verify that it's a "valid" BLS signature which was where the regression is. As we previously use signing root instead of signature (different byte lengths)